File: PLD1_InputProcessOutput/EX7_milesPerGallon/milesPerGallon.cpp
#include <iostream>
using namespace std;

int main()
{
    int MPG;
    int milesDriven;
    int gallons;

    cout << "Miles Driven :";
    cin >> milesDriven;
    cout << "Gallons Gas  :";
    cin >> gallons;

    MPG = milesDriven / gallons;

    cout << "Miles-per-Gallon : " << MPG << endl;

    return 0;
}

File: PLD1_InputProcessOutput/EX7_milesPerGallon/milesPerGallon.php
<?php
$milesDriven = readline("Miles Driven :");
$gallons = readline("Gallons Gas :");

$MPG = $milesDriven /  $gallons;
$MPG = number_format((float)$MPG, 1, '.', '');


echo "Miles-per-Gallon :" . $MPG;
